src/components/WebexLocalMedia/WebexLocalMedia.scss

```diff
@@ -2,16 +2,30 @@
   display: flex;
   justify-content: center;
   align-items: center;
+
   height: 100%;
+  min-height: 7.5rem; // Matches avatar's tallest height since transforms are just visual
```

Comment:
The min height matches the "tallest"? height of the avatar? Shouldn't it match the smallest and then transform?

Reply:
So what this means is that the smallest you can shrink local media (height-wise) is as tall as the biggest size an avatar inside _can_ get.
do you have suggestions on how to rephrase that comment?

Reply:
> So what this means is that the smallest you can shrink local media (height-wise) is as tall as the biggest size an avatar inside _can_ get.
> do you have suggestions on how to rephrase that comment?

`the smallest you can shrink local media (height-wise) is as tall as the biggest size an avatar inside can get.` 😆

Reply:
Do we need a `min-width` as well?

Reply:
> > So what this means is that the smallest you can shrink local media (height-wise) is as tall as the biggest size an avatar inside _can_ get.
> > do you have suggestions on how to rephrase that comment?
> 
> `the smallest you can shrink local media (height-wise) is as tall as the biggest size an avatar inside can get.` 😆

so wordy! lol

Reply:
> Do we need a `min-width` as well?

I didn't set one because the avatar sizing is based on the width, but I could set it to the same thing as the height